trestle/core/generators.py

```diff
@@ -94,22 +96,38 @@ def generate_sample_value_by_type(
     elif type_ is pydantic.networks.AnyUrl:
         # TODO: Cleanup: this should be usable from a url.. but it's not inuitive.
         return pydantic.networks.AnyUrl('https://sample.com/replaceme.html', scheme='http', host='sample.com')
+    elif type_ == Any:
+        # Return empty dict - aka users can put whatever they want here.
+        return {}
     else:
         raise err.TrestleError(f'Fatal: Bad type in model {type_}')
 
 
-def generate_sample_model(model: Union[Type[TG], List[TG], Dict[str, TG]]) -> TG:
-    """Given a model class, generate an object of that class with sample values."""
-    # FIXME: Typing is wrong.
-    # TODO: The typing here is very generic - which may cause some pain. It may be more appropriate to create a wrapper
-    # Function for the to level execution. This would imply restructuring some other parts of the code.
+def generate_sample_model(
+    model: Union[Type[TG], List[TG], Dict[str, TG]], optional: bool = False, depth: int = -1
+) -> TG:
+    """Given a model class, generate an object of that class with sample values.
+
+    Can generate optional variables with an enabled flag. Any array objects will have a single entry injected into it.
+
+    Note: Trestle generate will not activate recursive loops irrespective of the depth flag.
+
+    Args:
+        model: The model type provided. Typically for a user as an OscalBaseModel Subclass.
+        optional: Whether or not to generate optional fields.
```

Comment:
for clarity might be good to call this include_optional instead of optional because it looks a little odd

Reply:
+1

Reply:
Done.